phpBB/adm/style/admin.css

```diff
@@ -1670,6 +1670,112 @@ fieldset.submit-buttons legend {
 	}
 }
 
+/* Mentions and mention dropdown
+---------------------------------------- */
+.mention {
+	font-weight: bold;
+}
+
+.mention-container { /* mention-container */
+	text-align: left;
+	background-color: #ffffff;
+	border-radius: 2px;
+	box-shadow:
+		0 3px 1px -2px rgba(0, 0, 0, 0.2),
+		0 2px 2px 0 rgba(0, 0, 0, 0.14),
+		0 1px 5px 0 rgba(0, 0, 0, 0.12);
+	position: absolute;
+	z-index: 999;
+	overflow: auto; /* placed here for list to scroll with arrow key press */
+	max-height: 200px;
+	transition: all 0.2s ease;
+}
+
+.rtl .mention-container { /* mention-container */
+	text-align: right;
+}
+
+.mention-container ul { /* mention-list */
+	margin: 0;
+	padding: 0;
+	list-style-type: none;
+}
+
+.mention-media {
+	color: #757575;
+	display: inline-flex;
+	flex-shrink: 0;
+	justify-content: center;
+	align-items: center;
+	margin-right: 8px;
+	margin-left: 0;
+}
+
+.rtl .mention-media {
+	margin-right: 0;
+	margin-left: 16px;
+}
+
+.mention-media-avatar {
+	width: 40px;
+	height: 40px;
+}
+
+.mention-media-avatar img { /* TODO: reconsider for the future design with 1:1 ratio images */
```

Comment:
we do not need this the avatar should be using the global avatar phpbb class which should already be doing this...

Reply:
I don't think we have such a clear definition yet. There are only specific declarations for `.postprofile .avatar`

Reply:
What is the height even doing though?

Reply:
Ensuring the vatar is being displayed ;)

Reply:
yes prosilver is not using it yet but it can and should. We reworked all avatars to always have the class we just never reworked all the random css targeting them. The container should be setting the size for all of them which is why this is needed. otherwise, every avatar used everywhere would need specified HTML attributes for the height/width. I would suggest removing it here for adding `.avatar` in common.css after `.rowbg`

Reply:
given the changes to the html this should not require the img tag anymore